packages/x-components/tests/e2e/common/common-steps.spec.ts

```diff
@@ -248,19 +248,73 @@ Then(
   (key: string, value: string) => {
     cy.wait('@interceptedResults')
       .its('request.body')
-      .then(JSON.parse)
-      .should('have.property', key, value === 'default' ? '' : value);
+      .should((body: string) => {
+        expect(JSON.parse(body)).to.have.property(key, value);
```

Comment:
Cypress suggests avoiding the `.then` call for transforming values due to retryability. I don't think that really matters here  as we are using a `wait` command ( I thought it did), but now that I see it I like it.

https://docs.cypress.io/guides/core-concepts/retry-ability#Incorrectly-waiting-for-values